helm/methode-content-collection-mapper/values.yaml

```diff
@@ -13,5 +13,7 @@ env:
     opts: "-Xms128m -Xmx128m -XX:+UseG1GC -server"
   document_store_api_url: ""
 resources:
+  requests:
+    memory: 128Mi
```

Comment:
The lowest average memory usage I see is around 200 MB, and the most common average I see is 285 MB. I would increase this a bit maybe.